_posts/2017-05-01-five-lessons-learned.markdown

```diff
@@ -0,0 +1,80 @@
+There will never be a shortage of learning opportunities at this level,
+or so it feels.
+
+With that, here are the top five things I have learned in my time as a Director:
+  
+## Continuous Learning Is a Fantastic Investment
+Continuous learning is relatively new as an HR bonus but its basic premise has
+been around for a while. Here are some things we do that help with continuous
+learning:
+- Office hours for experimentation
+- Conference/workshop budgets for exposure to new ideas and new tools
+- Code review
+- Pair programming
+- Mentoring
+- Technical book a month club (not required reading, but we buy a technical
+book every month)
+
+Thus far the response to the above has led to highly engaged and highly
+motivated individuals that continue to apply new tools to new situations and
+are continually creating more sustainable and maintainable software. [Here]()
+is a blog post by one of our engineers speaking about Continuous Learning.
+
+## Breaking Down Knowledge Silos
+As teams grow and scale, knowledge silos tend to form around services,
+codebases, and processes. Certain engineers that build/maintain codebases
+end up with all of the knowledge around how that piece of infrastructure
+interacts in the ecosystem. It's important to disseminate this information as
+succinctly as possible. Other than creating documentation in the codebase, we
+have found that using [discourse](https://www.discourse.org/)
+and documenting the following has helped immensely:
+- Business Processes
+- Code Paths
+- Post mortems
+
+Each one of those deserves its own blog post, but suffice to say it helps
+onboarding new engineers to understand how each app works in the ecosystem and
+what it's role is.  
+
+## Perceived Failure is an Actual Blessing
+Failure is defined differently from person to person. To some, it's about using
+the wrong software pattern in the wrong circumstance. To others, its deploying
+bad code to production. Whatever the case may be, it's important that teams
+understand its absolutely ok to make mistakes. From experience, it seems that
+most engineers learn the most from their missteps.
+
+## 1-1 Meetings are Indispensible
+I never really understood the importance of the 1-1 while being an engineer. You
+have to take time out of your day and spend 30 minutes with your boss addressing
+things ranging from emotions to tasks. Now as a director, the 1-1 is one of
+the most important set of meetings you will have on a recurring basis. It gives
+you insight into exactly how best to step up and fill in the gaps that the team
+needs, which changes constantly. Most importantly, it sets your team up to be
+successful as you support their needs.
+  
+## The Most Important Aspect of a Director's Job is...
+Listening. I won't expound much on this as many other highly qualified
+individuals have covered this topic at length. Everyone has pain points while
+working at any company. It is important to make sure that individuals feel heard
```

Comment:
It is important to make sure ~~that~~ individuals feel heard and feel ~~that they are~~ supported within their organization.